packages/flutter/test/widgets/custom_painter_test.dart

```diff
@@ -401,7 +401,7 @@ void _defineTests() {
     semantics.dispose();
   });
 
-  testWidgets('Supports all flags', (WidgetTester tester) async {
+/*  testWidgets('Supports all flags', (WidgetTester tester) async {
```

Comment:
Instead of commenting this out, the test should be skipped with the `skip` argument: https://master-api.flutter.dev/flutter/flutter_test/testWidgets.html

There also should be a comment with a TODO indicating when to re-enable this test.